src/components/Brewery.js

```diff
@@ -16,6 +16,25 @@ export default class Brewery extends Component {
     )
   }
 
+  displayLocation(street, city, state) {
+    // format for Google Maps
+    var street = street.replace(/ /g,"+")
+    var city = city.replace(/ /g,"+")
+    var state = state.replace(/ /g,"+")
+    var url = "https://www.google.com/maps/place/"+street+","+city+","+state
+    return (
+      <a
```

Comment:
If you run `npm run lint` you'll see a slew of errors. These are very easy to correct as you are redeclaring declared variables (street, city, state) and `const` is preferred over `var` as the street, city, and state are constants, meaning they don't get changed after declaration.

To make this code more reusable I would advise moving the following chunk into a separate function:
```js
    var street = street.replace(/ /g,"+")
    var city = city.replace(/ /g,"+")
    var state = state.replace(/ /g,"+")
    var url = "https://www.google.com/maps/place/"+street+","+city+","+state
```
something like this:

```js
  formatAddressUrl(address) {
    const formattedAddress = address
      .map(location => location.replace(/ /g, '+'))
      .join()

    return `https://www.google.com/maps/place/${formattedAddress}/`
  }
```

`address would be an array `[street, city, state]` which be the parameter for the `displayLocation(address)` function.